README.md

```diff
@@ -1,13 +1,51 @@
-After creating the database for this application, you must enable two
-PostgreSQL extensions:
+# SimplyE Library Registry
 
+A geographic search engine for matching people to the libraries that serve them.
+
+## Installation
+
+Because the Registry runs in a Docker container, the only required software is [Docker Desktop](https://www.docker.com/products/docker-desktop). The database and webapp containers expect to be able to operate on ports 5432 and 80, respectively--if those ports are in use already you may need to amend the `docker-compose.yml` file to add alternate ports.
+
+_Note: If you would like to use the `Makefile` commands you will also need `make` in your `PATH`. They're purely convenience methods, so it isn't strictly required. If you don't want to use them just run the commands from the corresponding task in the `Makefile` manually. You can run `make help` to see the full list of commands._
+
+### Building the Images
+
+Local development uses two Docker images and one persistent Docker volume (for the PostgreSQL data directory). To create the base images:
+
+```shell
+make build
+```
+
+## Usage
+
+### Running the Containers
+
+You can start up the local compose cluster in the background with:
+
+```shell
+make up
 ```
-CREATE EXTENSION postgis;
-CREATE EXTENSION fuzzystrmatch;
```

Comment:
I don't think you can remove this information from the human-readable documentation. In a non-development environment the database will be configured separately and this step will still need to be run manually, right? At least we need a pointer to the script that runs this for the Docker database image.

Reply:
I just put non-Docker install instructions into the README, in b3bd736.